README.md

```diff
@@ -369,8 +352,8 @@ A route should have been deployed via template for HTTPS access on the MIQ pod
 
 ```bash
 $oc get routes
-NAME       HOST/PORT                       PATH      SERVICE            TERMINATION   LABELS
-manageiq   miq.apps.e2e.bos.redhat.com             manageiq:443-tcp   passthrough   app=manageiq
+NAME      HOST/PORT                              PATH      SERVICES   PORT      TERMINATION     WILDCARD
+httpd     miq-dev.apps.bos.redhat.com                      httpd      http      edge/Redirect   None
```

Comment:
Oh didn't see this before, I feel like we should be using `example.com` for our documentation.